Tests/UKHO.MaritimeSafetyInformation.Web.UnitTests/Services/FileShareServiceTest.cs

```diff
@@ -125,5 +125,40 @@ public void WhenFSSDownloadFileAsyncThrowsException_ThenShouldExecuteCatch()
             Task<Stream> result = _fileShareService.FSSDownloadFileAsync("", "", "", CorrelationId);
             Assert.IsTrue(result.IsFaulted);
         }
+
+        [Test]
+        public void WhenFSSDownloadZipFileIsCalled_ThenShouldReturnByteArray()
+        {
+            string batchId = Guid.NewGuid().ToString();
+            const string fileName = "Daily 16-05-22.zip";
+
+            IResult<Stream> stream = new Result<Stream>
+            {
+                Data = new MemoryStream(Encoding.UTF8.GetBytes("test stream"))
+            };
+
+            _fileShareServiceConfig.Value.BaseUrl = "https://filesqa.admiralty.co.uk";
+
+            A.CallTo(() => _fileShareApiClient.DownloadZipFileAsync(batchId, CancellationToken.None)).Returns(stream);
+
+            Task<Stream> result = _fileShareService.FSSDownloadZipFile(batchId, fileName, FakeAccessToken, CorrelationId);
+
+            Assert.IsInstanceOf<Task<Stream>>(result);
+        }
+
+        [Test]
+        public void WhenFSSDownloadZipFileThrowsException_ThenShouldExecuteCatch()
```

Comment:
As per previous comment:
```suggestion
        public void WhenFSSDownloadZipFileAsyncThrowsException_ThenShouldExecuteCatch()
```